integration-tests/src/test/resources/performance-features/continuously-create-bridges.feature

```diff
@@ -0,0 +1,130 @@
+                    }
+                  headers:
+                    content-type: application/json
+                    authorization: Bearer ${accessToken}
+                  handler:
+                    body:
+                      json:
+                        query: .id
+                        toVar: bridgeId
+            - create-poll:
+              # Wait until the Bridge is in either in ready or failed state
+              - httpRequest:
+                  endpoint: manager
+                  GET: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    content-type: application/json
+                    authorization: Bearer ${accessToken}
+                    cache-control: no-cache
+                  handler:
+                    body:
+                      json:
+                        query: .status
+                        toVar: bridgeStatus
+              - breakSequence:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: ready
+                  onBreak:
+                    newSequence:
+                      sequence: delete
+                      forceSameIndex: true
+              - fail:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: failed
+                  message: "Bridge creation failed"
+              - thinkTime:
+                  duration: 5s
+              - restartSequence
+            - delete:
+              # Delete Bridge
+              - httpRequest:
+                  endpoint: manager
+                  DELETE: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    authorization: Bearer ${accessToken}
+            - delete-poll:
+              # Wait until the Bridge is in either deleted or in failed state
+              - httpRequest:
+                  endpoint: manager
+                  GET: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    content-type: application/json
+                    authorization: Bearer ${accessToken}
+                    cache-control: no-cache
+                  handler:
+                    status:
+                      store:
+                        toVar: reponseStatus
+                    body:
+                      json:
+                        query: .status
+                        toVar: bridgeStatus
+              - breakSequence:
+                  intCondition:
+                    fromVar: reponseStatus
+                    equalTo: 404
+              - fail:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: failed
+                  message: "Bridge deletion failed"
+              - thinkTime:
+                  duration: 5s
+              - restartSequence
+      """
+    Then run benchmark "rhose-create-bridges" within 15 minutes
```

Comment:
if maxDuration is set to 60 minutes for the perf scenario, does it make sense to wait for only 15 minutes here?

Reply:
That was a copy paste leftover, will fix it.
Makes me think, would it have sense to merge this step with the benchmark creation? Is there a scenario when we would want to create a scenario without executing it right away?

Reply:
true... I can't think of any reason why we would want to have a benchnmark scenario created and not run it right away - would make sense to merge both from my POV